edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_dashboard.scss

```
@@ -1,436 +1,326 @@
-  }
+p#track-info {
+  padding: 20px;
 }
 
-.dashboard .main-container .my-courses
-.course .details .course-actions
-.wrapper-action-more div.actions-dropdown {
-  min-width: 108px;
-  width:100%;
-  right: 0;
-  top: 50px;
+div.modal-content .modal-header button.close {
+  background: transparent;
+  float: right;
+  font-size: 1.625rem;
+  border: 0;
+  box-shadow: none;
 }
 
-// Upgrade button style and verify button hide.
-.wrapper-messages-primary {
-
-  .message.is-shown {
-    padding: 0 0 10px 0;
-  }
-
-  //verify
-
-  .message.message-status {
-
-    .verification-reminder {
-      display: none;
-    }
-
-    .verification-cta {
-      display: none;
-    }
+.course-search-home .search-home-container {
+  padding: 10px;
 
+  .icon {
+    top: 18px;
+    right: 25px;
   }
 
-  //upgrade
-
-  .message.message-upsell {
-
-    .wrapper-extended .message-copy {
-      display: none;
-    }
-
-    .action-upgrade-container {
-      width: 100%;
-      margin: 0 !important;
-    }
+  #search-input {
+    width: 25%;
+    float: right;
 
-    .action-upgrade {
+    @include media-breakpoint-down(md) {
       width: 100%;
-      color: #003057;
-      max-width: 192px;
-      background-color: #D2DB0E !important;
-      }
-
-    .action-upgrade:hover {
-      background-color: #84BD00;
-    }
-
-    .wrapper-extended {
-      padding: 0;
     }
   }
 }
 
-
-// General styling
-
-.dashboard .main-container .my-courses
-.course .details .enter-course {
-  color: $primary;
-  background: $primary;
-}
-
-.wrapper-messages-primary .message.message-upsell
-.action-upgrade span {
-  color: $primary;
-}
-
-div.course-caption p.localized-datetime {
-  display: none;
+// Content layout.
+.view-dashboard .content-wrapper {
+  padding: 0;
 }
 
-section[role=courses] div.container.no-border {
-  max-width: 2100px;
+.view-dashboard .container {
+  max-width: none;
   min-height: 77vh;
+  padding: 10px 15px;
 
-  header {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: center;
-    padding: 20px;
+  @include media-breakpoint-up(xl) {
+    padding: 30px 55px;
+  }
+}
 
-    p {
-      width: 100%;
-      text-align: center;
-      margin-bottom: 40px;
-    }
+// Empty message.
+.view-dashboard .empty-dashboard-message {
+  color: $black;
+  background: $dashboard-contrast;
+  border: none;
+  padding: 60px 0;
 
-    a.btn.btn-lg.btn-default,
-    a.btn.btn-lg.btn-default:hover {
-      color: $inverse;
-    }
+  p {
+    color: $black;
+    font-size: 1.313rem;
   }
 
-  h1 {
-    font: normal 2.5em Playfair Display;
-    margin-bottom: 30px;
-    text-align: left;
-    font-family: inherit;
-    font-weight: lighter;
-  }
+  a.btn {
+    background: $primary-dark;
+    font-size: 0.75rem;
+    text-transform: uppercase;
+    padding: 12px 20px;
 
-  @include media-breakpoint-down(md) {
-    h1 {
-      display: none;
+    &:hover,
+    &:focus {
+      background: linear-gradient(0deg, rgba(0, 0, 0, 0.1), rgba(0, 0, 0, 0.1)), $primary-dark;
```

```suggestion
      background: #991f34;
```

Would you consider using a solid color to allow transitions?

---

OK, yes I used the linear-gradient has referenced in Figma but this is better, thanks!